modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackService.kt

```diff
@@ -303,8 +297,11 @@ open class PlaybackService : MediaBrowserServiceCompat(), CoroutineScope {
                             LogBuffer.i(LogBuffer.TAG_PLAYBACK, "stopForeground state: $state removing notification: $removeNotification")
                         }
 
-                        @Suppress("DEPRECATION")
-                        stopForeground(removeNotification)
+                        // When paused keep the notification otherwise remove it
+                        stopForeground(if (removeNotification) STOP_FOREGROUND_REMOVE else STOP_FOREGROUND_DETACH)
+                        if (removeNotification) {
+                            notificationManager.cancel(Settings.NotificationId.PLAYING.value)
```

Comment:
If you long press on the mini player and choose "Close and clear Up Next" before this change there would be a blank notification without any controls left in the notification drawer.